### ISSUE TITLE
Broken links in book

### ISSUE THREAD

**Issue by michaelsproul** [sigp/lighthouse#2889]
## Description

Running [`linkcheck`](https://github.com/filiph/linkcheck) on the Lighthouse Book reveals a number of broken links:

```
$ linkcheck https://lighthouse-book.sigmaprime.io/
Done crawling.                   

https://lighthouse-book.sigmaprime.io/
- (151:42) 'local te..' => https://lighthouse-book.sigmaprime.io/local-testnets.html (HTTP 403)
- (151:42) 'local te..' => https://lighthouse-book.sigmaprime.io/local-testnets.html (HTTP 403)

https://lighthouse-book.sigmaprime.io/faq.html
- (191:14) '/beacon/..' => https://lighthouse-book.sigmaprime.io/http/beacon.html#beaconvalidatorsall (HTTP 403)
- (196:0) '/beacon/..' => https://lighthouse-book.sigmaprime.io/http/beacon.html#beaconvalidatorsall (HTTP 403)
- (260:54) 'here.' => https://lighthouse-book.sigmaprime.io/installation-source.html#updating-lighthouse (HTTP 200 but missing anchor)

https://lighthouse-book.sigmaprime.io/intro.html
- (151:42) 'local te..' => https://lighthouse-book.sigmaprime.io/local-testnets.html (HTTP 403)

https://lighthouse-book.sigmaprime.io/pi.html
- (154:15) 'Ubuntu D..' => https://lighthouse-book.sigmaprime.io/installation.html#dependencies-ubuntu (HTTP 200 but missing anchor)

https://lighthouse-book.sigmaprime.io/print.html
- (152:42) 'local te..' => https://lighthouse-book.sigmaprime.io/local-testnets.html (HTTP 403)
- (593:15) 'Ubuntu D..' => https://lighthouse-book.sigmaprime.io/installation.html#dependencies-ubuntu (HTTP 200 but missing anchor)
- (2145:7) 'Global V..' => https://lighthouse-book.sigmaprime.io/validator-inclusion.html#consensusglobal_votes (HTTP 200 but missing anchor)
- (3321:52) 'explicit..' => https://lighthouse-book.sigmaprime.io/subscribe-api (HTTP 403)
- (3686:14) '/beacon/..' => https://lighthouse-book.sigmaprime.io/http/beacon.html#beaconvalidatorsall (HTTP 403)
- (3691:0) '/beacon/..' => https://lighthouse-book.sigmaprime.io/http/beacon.html#beaconvalidatorsall (HTTP 403)
- (3755:54) 'here.' => https://lighthouse-book.sigmaprime.io/installation-source.html#updating-lighthouse (HTTP 200 but missing anchor)

https://lighthouse-book.sigmaprime.io/redundancy.html
- (211:52) 'explicit..' => https://lighthouse-book.sigmaprime.io/subscribe-api (HTTP 403)

https://lighthouse-book.sigmaprime.io/validator-inclusion.html
- (213:7) 'Global V..' => https://lighthouse-book.sigmaprime.io/validator-inclusion.html#consensusglobal_votes (HTTP 200 but missing anchor)

Errors. Checked 4448 links, 83 destination URLs (72 ignored), 3 have errors, 6 have warnings.
```

## Version

Book as of 1 January 2022

## Steps to resolve

Correct broken links by working out what they should point to instead. If feeling ambitious, try running `linkcheck` on a locally-hosted version of the book under CI.

**Comment by ackintosh** [comment]
Memo: 
- [super linter](https://github.com/github/super-linter) doesn't check links
- [mdbook linkcheck](https://github.com/Michael-F-Bryan/mdbook-linkcheck) doesn't check anchors

**Comment by michaelsproul** [comment]
Closed by #2918 and #2912, thanks @ackintosh!
